src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -434,6 +419,11 @@ int main(int argc, char **argv)
             usage();
             exit(EXIT_SUCCESS);
             break;
+        default:
+            fprintf(stderr, "Unknown option: %c\n", opt);
```

Comment:
This code does not follow getopt(3) API AFAICT -- printed `opt` does _not_ store the name of an unknown option in this context. I have not checked all of the details, but I would start with testing getopt(3) build-in error reporting:

```suggestion
```

or

```suggestion
            // getopt() emits error message to stderr
```

If that does not work, look for "getopt() can detect two kinds" in getopt(3) [documentation](https://www.man7.org/linux/man-pages/man3/getopt.3.html).

Reply:
Done

Reply:
> Done.

Not done (including/or not pushed).

Reply:
Sigh.. sorry.
Now done for real